src/components/App/App.js

```diff
@@ -12,11 +12,36 @@ class App extends Component {
     }
   }
 
+  updateGender = (vibe, entity) => {
+    this.setState({ currentGender: `${vibe} ${entity}`})
+  }
+
+  componentDidMount = async () => {
+    try {
+      const fetchedAnimals = await getWords('animal');
```

Comment:
Great work here. Looks really clean and easy to read. Way to think about readability and structure!